NUAttribute.js

```diff
@@ -57,7 +57,8 @@ export default class NUAttribute extends NUObject {
             userlabel: obj.userlabel,
             isInternal: obj.isInternal,
             minValue: obj.minValue,
-            maxValue: obj.maxValue
+            maxValue: obj.maxValue,
+            isTransient: obj.isTransient
```

Comment:
you can consider having this as `!!obj.isTransient` like the other booleans, will ensure a boolean value even if undefined

Reply:
Actually we should keep undefined. Not sure why the others are done this way.It's important to know that this is not defined/null or the boolean is set.